risc0/zkvm/src/exec/mod.rs

```diff
@@ -28,7 +28,9 @@ mod tests;
 
 use std::{cell::RefCell, fmt::Debug, io::Write, mem::take, rc::Rc};
 
+use ::bonsai_sdk::alpha::SdkErr;
```

Comment:
You could probably remove this line and replace it with `use bonsai_sdk::SdkErr` after you alias bonsai_sdk::alpha below.